packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,160 @@
+default.target=.
+
+## all templates to apply on target deployment env
+nuxeo.template.includes=default,s3binaries,postgresql,explorer-sitemode,explorer-virtualadmin
+# TODO: check logs config with 'docker' or 'docker-json' template
+
+## explorer confs mapping
+org.nuxeo.apidoc.apidocAdmin.password=${env:ORG_NUXEO_APIDOC_APIDOCADMIN_PASSWORD, ""}
+
+
+## nuxeo.conf mappings
+
+nuxeo.log.dir=${env:NUXEO_LOG_DIR:/var/log/nuxeo}
+nuxeo.pid.dir=${env:NUXEO_PID_DIR:/var/run}
+nuxeo.tmp.dir=${env:NUXEO_TMP_DIR:/tmp}
```

Comment:
Should be handled by the parent image, what do you think @troger?

Reply:
i was indeed not sure about these (note i took the "recommended" default values, maybe that was not a good idea either in this context)

Reply:
Yes, those are clearly not needed and are configured in the Nuxeo Docker image.